registry/server/appRoutes/routes/common.ts

```diff
@@ -0,0 +1,20 @@
+import db from '../../db';
```

Comment:
`common` is too generic name for a file, please name it `routesRepository`

Reply:
done